recipes/godot_headers/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "3.0":
+    url: "https://github.com/godotengine/godot_headers/archive/3.0.zip"
+    sha256: "d73f6dc4d060cadc6c37a82f162cbc0e5ba91db6998ad4d25bbd433184723a00"
+  "3.1":
+    url: "https://github.com/godotengine/godot_headers/archive/3.1.zip"
+    sha256: "95ccd7e8b5809f12f048e82f52d59fee054d8e0ed22fd0b40ce6feb00dc09d6b"
```

Comment:
Do we need 3.0 and 3.1?

Reply:
I kept them for completeness, however i have no issue with dropping them.

Reply:
I think we should start with the last version and if people really need a previous version, then can request it or add it themselves.


The release page is short: https://github.com/godotengine/godot_headers/releases